app/views/users/show.html.erb

```diff
@@ -46,6 +46,13 @@
     </span>
   <% end %>
 <% end %>
+<% if current_user.administrator? %>
+  <%= link_to reset_password_user_path(@user), method: :post do %>
+    <span class="btn">
+      Reset Password
```

Comment:
Can you change this to something like "Change Password for User?"

Reply:
Okay, address in recent push